internal/config/config.go

```diff
@@ -79,6 +79,7 @@ func Load(ctx context.Context, l *zap.SugaredLogger) {
 	}
 
 	Logger = l
```

Comment:
why have both the exported const and logger on the struct? 

ie, there is the `config` pkg's Logger, `config.Logger` and the `Config` type's Logger. Do we need both?

Reply:
you made me curious, in https://github.com/packethost/aws-s3-proxy/blob/master/internal/s3/s3.go#L27 that's the only usage of the global logger reference. I think I'll just change it to the config.Cfg.Logger instead